pkg/apiserver/apiserver.go

```diff
@@ -272,32 +272,44 @@ func write(statusCode int, apiVersion string, codec runtime.Codec, object runtim
 		io.Copy(writer, out)
 		return
 	}
-	writeJSON(statusCode, codec, object, w)
+	writeJSON(statusCode, codec, object, w, isPrettyPrint(req))
+}
+
+func isPrettyPrint(req *http.Request) bool {
+	pp := req.URL.Query().Get("pp")
```

Comment:
just pushed...  Sorry, I was 'done-ing' comments as I came to them, and I needed to add the swagger docs you asked for...